### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Weblate](https://hosted.weblate.org/projects/kimai/messages/) for Kimai - Time tracking/Main application.

Current translation status:

Weblate translation status

### CODE REVIEW

File: translations/email.zh-Hans.xlf [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+  <file source-language="en" target-language="zh-Hans" datatype="plaintext" original="email.en.xlf">
+    <body>
+      <trans-unit id="button_copy_url_to_browser">
+        <source>button_copy_url_to_browser</source>
+        <target>如果该按钮无效，请复制 &amp; 以下URL并粘贴到浏览器中: %url%</target>
+      </trans-unit>
+      <trans-unit id="automated_email_dont_answer">
+        <source>automated_email_dont_answer</source>
+        <target/>
```
Comment (kevinpapst) [inline translations/email.zh-Hans.xlf:11]: missing

File: translations/reporting.pl.xlf [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+  <file source-language="en" target-language="pl" datatype="plaintext" original="reporting.en.xlf">
+    <body>
+      <trans-unit id="reporting.title">
+        <source>reporting.title</source>
+        <target></target>
```
Comment (kevinpapst) [inline translations/reporting.pl.xlf:7]: missing

**Comment by psycowby** [comment]
Hello Kevin, i am working on translation to the Czech language by weblate. Hope it helps :).

**Comment by kevinpapst** [comment]
Awesome @psycowby - thank you for your support!!!

**Comment by psycowby** [comment]
finish.

**Pull request merged**